docs/intro/acquiring_data/index.rst

```diff
@@ -4,14 +4,534 @@
 Acquiring Data
 **************
 
-This section of the guide introduces ways to obtain different kind of solar data from different places.
+This section of the tutorial introduces ways to obtain different kind of solar data from different places.
+The main tutorial below focuses on ``Fido``, which is a generic search interface that sunpy provides.
+There are some data-provider specific tutorials that you might want to visit after going through this tutorial:
 
 .. toctree::
     :maxdepth: 1
 
-    fido
     jsoc
     hek
     helioviewer
     sample-data
     database
+
+
+This guide outlines how to search for and download data using the `~sunpy.net.Fido` interface for search and download.
```

Comment:
```suggestion
This tutorial outlines how to search for and download data using the `~sunpy.net.Fido` interface for search and download.
```

maybe guide is the correct phrasing here, but maybe to be consistent tutorial would be better?